docs/en/observability/monitor-gcp.asciidoc

```diff
@@ -0,0 +1,457 @@
++
+[source,bash]
+----
+./filebeat keystore create
+echo -n "<Your Deployment Cloud ID>" | ./filebeat keystore add CLOUD_ID --stdin
+----
+
+. To store logs in {es} with minimal permissions, create an API key to send
+data from {filebeat} to {ess}. Log into Kibana (you can do so from the Cloud
+Console without typing in any permissions) and select *Management* -> *Dev
+Tools*. Then, send the following request:
++
+[source,console]
+----
+POST /_security/api_key
+{
+  "name": "filebeat-monitor-gcp",
+  "role_descriptors": {
+    "filebeat_writer": {
+      "cluster": ["monitor", "read_ilm", "cluster:admin/ingest/pipeline/get", "cluster:admin/ingest/pipeline/put"],
+      "index": [
+        {
+          "names": ["filebeat-*"],
+          "privileges": ["view_index_metadata", "create_doc"]
+        }
+      ]
+    }
+  }
+}
+----
++
+[NOTE]
+----
+{filebeat} needs two special cluster permissions
+`cluster:admin/ingest/pipeline/get` and `cluster:admin/ingest/pipeline/put`.
+----
+
+. The response contains an `api_key` and an `id` field, which can be stored in
+the {filebeat} keystore in the following format: `id:api_key`.
++
+[source,bash]
+----
+echo -n "IhrJJHMB4JmIUAPLuM35:1GbfxhkMT8COBB4JWY3pvQ" | ./filebeat keystore add ES_API_KEY --stdin
+----
++
+[NOTE]
+=====
+Make sure you specify the `-n` parameter; otherwise, you will have
+painful debugging sessions, due to adding a newline at the end of
+your API key.
+=====
+
+. To see if both settings have been stored, run `./filebeat keystore list`.
+
+. Then, configure {filebeat} to output to {ess}. Edit the `filebeat.yml`
+configuration file and add the following lines to the end of the file.
++
+[source,yml]
+----
+cloud.id: ${CLOUD_ID}
+output.elasticsearch:
+  api_key: ${ES_API_KEY}
+----
+
+. Finally, test if the configuration is working. If it is not working,
+verify if you used the right credentials and add them again.
++
+[source,bash]
+----
+./filebeat test output
+----
+
+Now that the output is working you are going to setup the input ({gcp}).
+
+[discrete]
+=== Step 5: Configure {filebeat} Google Cloud module
+
+In order to collect logs from Google Cloud Platform, use the
+https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-gcp.html[Google Cloud Platform]
+module. This module periodically fetches logs that have been exported from
+Stackdriver to a Google Pub/Sub topic sink. There are three available filesets:
+`audit`, `vpcflow`, `firewall`. This tutorial covers the `audit` fileset.
+
+. First, you need to configure {gcp} to export logs to a Pub/Sub topic. You can
+do that by navigating to the *Logs Router* page (use the search bar):
++
+image::monitor-gcp-navigate-logs-router.png[Navigate to Logs Router page]
++
+Then, click on *create sink* to set up the logs routing sink.
+Set *sink name* as `monitor-gcp-audit-sink`. Select the *Cloud Pub/Sub topic* as the
+*sink service* and *Create new Cloud Pub/Sub topic* named `monitor-gcp-audit`.
```

Comment:
From the screenshot, I wasn't too sure where you `Create new Cloud Pub/Sub topic named monitor-gcp-audit`. Is this under Sink destination?

Reply:
Yes that would be great. Thanks!